src/apps/api/utils.py

```diff
@@ -1,18 +1,21 @@
 __author__ = 'felix.shaw@tgac.ac.uk - 20/01/2016'
 
 from common.lookup.lookup import API_RETURN_TEMPLATES
-from common.schemas.utils import data_utils as d_utils
-from common.schema_versions.lookup.dtol_lookups import GDPR_SENSITIVE_FIELDS, STANDARDS, STANDARDS_MAPPING_FILE_PATH
+from common.schemas.utils.data_utils import get_copo_schema
 from django.http import HttpResponse
 from django_tools.middlewares import ThreadLocal
-from io import BytesIO
+from .views.mapping import get_mapped_result
 
 import bson.json_util as jsonb
 import json
 import pandas as pd
 import shortuuid
 import uuid
-import zipfile
+
+def get_sensitive_fields(component):
+    schema = get_copo_schema(component)
+    sensitive_fields = [x['id'].split('.')[-1] for x in schema if x['is_sensitive']]
```

Comment:
Suggest to use x.get('is_sensitive', False)
As this function accepts any component.  It will be error if someone uses it for the component other than "sample".  If this function only works for "sample", make the function name as get_sample_sentitive_fields() and don't accept any component.